src/AutoRest.CSharp/Properties/launchSettings.json

```diff
@@ -432,10 +432,6 @@
       "commandName": "Project",
       "commandLineArgs": "--standalone $(SolutionDir)\\test\\TestProjects\\MgmtPropertyChooser\\src\\Generated -n"
     },
-    "MgmtReferenceTypes": {
-      "commandName": "Project",
-      "commandLineArgs": "--standalone $(SolutionDir)\\test\\TestProjects\\MgmtReferenceTypes\\src\\Generated -n"
-    },
```

Comment:
well looks like we have to delete this test project, which is unfortunate.
In the future we need to pay more attention on the regen PR for resourcemanager, because there are a few features that work differently when the IsArmCore configuration is true, and we now do not have test cases for them.

Reply:
Yes, if we use `IsArmCore` instead of the namespace, we can keep them, but we prefer namespace for accuracy.